view/frontend/templates/script/sentry.phtml

```diff
@@ -2,11 +2,17 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 /** @var \JustBetter\Sentry\Block\SentryScript $block */
 
-if (! $block->canUseScriptTag($block->getNameInLayout())) {
+if (!$block->canUseScriptTag($block->getNameInLayout())) {
     return;
 }
 ?>
 
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Promise" crossorigin></script>
-<script src="https://browser.sentry-cdn.com/4.5.3/bundle.min.js" crossorigin></script>
-<script>Sentry.init({ dsn: '<?= $block->escapeUrl(trim($block->getDSN())) ?>' });</script>
+<script src="https://browser.sentry-cdn.com/<?= \JustBetter\Sentry\Block\SentryScript::CURRENT_VERSION;?>/bundle.min.js"
+        integrity="sha384-lowBFC6YTkvMIWPORr7+TERnCkZdo5ab00oH5NkFLeQUAmBTLGwJpFjF6djuxJ/5" crossorigin="anonymous">
```

Comment:
Placing the integrity here feels like it's defeating the purpose of the `CURRENT_VERSION` constant since when you want to change the version you'll have to update the integrity anyways so you might as well do both in the template.

I'd either get rid of the integrity so you'll only have to change the one constant or add another constant storing the integrity.

Reply:
Good catch! Changed it.